thenewboston/utils/network.py

```diff
@@ -35,11 +35,11 @@ def validate_response(response):
     Validate status code
     Return response as Python object
     """
-
-    data = response.json()
-
+    
     if response.status_code >= 400:
-        err = f'status_code:{response.status_code} - {data}'
+        err = f'status_code:{response.status_code}'
```

Comment:
i'd suggest to keep the response, but provide it as a raw test
`err = f'status_code:{response.status_code} {response.text}'

@shahraizali

Reply:
My only concern is size of this data like in this case its going to be huge text chunk, mainly because its running on debug true but still

Reply:
@vosi ?

Reply:
yeah, agree! updated PR.